content/posts/tekton-jx-orchestration.md

```diff
@@ -0,0 +1,953 @@
+taskrun.tekton.dev/jx-dcanadillas-petclinic-jenkin-lg4nw-3-kubectl-deploy-kdr9x    4h
+taskrun.tekton.dev/jx-dcanadillas-petclinic-jenkin-lg4nw-3-maven-build-x2cjr       4h
+taskrun.tekton.dev/jx-dcanadillas-petclinic-jx-mas-1-from-build-pack-x245x         10h
+taskrun.tekton.dev/jx-dcanadillas-petclinic-jx-mas-ndtfh-2-from-build-pack-9xkrq   27m
+
+NAME                                                          AGE
+pipeline.tekton.dev/dcanadillas-kube-environment-dc-d495w-7   23m
+pipeline.tekton.dev/dcanadillas-kube-environment-dc-hk95j-1   24m
+pipeline.tekton.dev/dcanadillas-kube-environment-dc-wm85w-6   10h
+pipeline.tekton.dev/dcanadillas-kube-environment-dc-wpqsp-1   10h
+pipeline.tekton.dev/jx-dcanadillas-petclinic-jenkin-lg4nw-3   4h
+pipeline.tekton.dev/jx-dcanadillas-petclinic-jx-mas-1         10h
+pipeline.tekton.dev/jx-dcanadillas-petclinic-jx-mas-ndtfh-2   27m
+
+NAME                                                             AGE
+pipelinerun.tekton.dev/dcanadillas-kube-environment-dc-d495w-7   23m
+pipelinerun.tekton.dev/dcanadillas-kube-environment-dc-hk95j-1   24m
+pipelinerun.tekton.dev/dcanadillas-kube-environment-dc-wm85w-6   10h
+pipelinerun.tekton.dev/dcanadillas-kube-environment-dc-wpqsp-1   10h
+pipelinerun.tekton.dev/jx-dcanadillas-petclinic-jenkin-lg4nw-3   4h
+pipelinerun.tekton.dev/jx-dcanadillas-petclinic-jx-mas-1         10h
+pipelinerun.tekton.dev/jx-dcanadillas-petclinic-jx-mas-ndtfh-2   27m
+
+NAME                                                                AGE
+pipelineresource.tekton.dev/dcanadillas-kube-environment-dc-d495w   23m
+pipelineresource.tekton.dev/dcanadillas-kube-environment-dc-hk95j   24m
+pipelineresource.tekton.dev/dcanadillas-kube-environment-dc-wm85w   10h
+pipelineresource.tekton.dev/dcanadillas-kube-environment-dc-wpqsp   10h
+pipelineresource.tekton.dev/jx-dcanadillas-petclinic-jenkin         4d
+pipelineresource.tekton.dev/jx-dcanadillas-petclinic-jenkin-lg4nw   4h
+pipelineresource.tekton.dev/jx-dcanadillas-petclinic-jenkin-rhg88   3d
+pipelineresource.tekton.dev/jx-dcanadillas-petclinic-jx-mas         10h
+pipelineresource.tekton.dev/jx-dcanadillas-petclinic-jx-mas-ndtfh   27m
+```
+
+## Some thoughts and final conclusions
+
+[Traditional Jenkins](https://jenkins.io/) has been for long one of the best CI/CD pipeline engines in terms of flexibility, standardization and adoption for most of DevOps environments. But new challenges for software delivery requires extremely scalable CI/CD architectures. That means today using Kubernetes as a powerful abstraction of the infrastructure and the use of cloud native platforms to decouple and scale CI/CD pipelines.
+
+Decoupling pipelines to scale is then one of the best approaches to build new modern applications and microservices. But it can be very hard to work on these decoupled objects to define a pipeline, which conceptually is a sequential and "monolithic" tasks execution.
```

Comment:
"Decoupling pipelines to scale is then one of the best approaches to build" to "Decoupling and scaling pipelines is one of the best approaches for building"